### ISSUE TITLE
Capstone: SQL

### ISSUE THREAD

**Issue by cviddy17** [cptgame446/sql-scratch-capstone-turn-in#1]
### Criteria 1: Report: Are conclusions clear and supported by evidence in all eight answers?
* _Score Level:_ 3/4 Meets Expectations
* _Comment(s):_ Conclusions directly supported evidence in this case.
 
### Criteria 2: Query Accuracy: Do queries do what they were supposed to do?
* _Score Level:_ 3/4 Meets Expctations
* _Comment(s):_ Queries returned correct results.
 
### Criteria 3: Query Formatting
* _Score Level:_ 3/4 Meets Expectations
* _Comment(s):_ Queries led to proper data, however, they could be commented better and subqueries could be more sufficiently used.
 
### Criteria 4: Understanding underlying concepts and terminology
* _Score Level:_ 3/4 Meets Expectations
* _Comment(s):_ Queries drove the data but analysis and further actions were not pursued.

 ### Overall Score: 12/16 Meets Expectations
Good job! You showed a demonstrable understanding of SQL and its applications. However, this was a very middle of the road project, which did not show a deep level of analysis and only a passable level of SQL. In the future, please put more focus on the presentation as a whole and make sure that all answers are clearly referenced in code.

**Comment by cptgame446** [comment]
Absolutely.  Thank you so much for the feedback.  I had a lot of fun learning and look forward to continue to practice and grow with SQL!
